### ISSUE TITLE
Candidate calendar field

### ISSUE THREAD

**Issue by Glynnethy** [Automattic/WP-Job-Manager#1388]
I am considering developing a Plugin myself that will add a calendar field to the standard WP Job Manager candidate field set to go alongside their resumes and applications on their dashboard (a calendar per candidate if required). Does anyone have any suggestions on how to approach this?

Many thanks.

#### Steps to Reproduce
1.
2.
3.

**Comment by tripflex** [comment]
Can you explain a use case scenario behind this so I can better understand what you're trying to do, then can point you in the right direction.  Sounds like you want to have a calendar on the dashboard for something ... but what will it be used for?  Displaying upcoming expiring jobs, dynamic or random info, etc?

**Comment by CVgeek** [comment]
I think it would be a great tool to havé a visual view of interviews and past interviews. I really like this idea.

-------- Original message --------

**Comment by Glynnethy** [comment]
Yes, as described, I am looking to add a calendar/organiser feature to the dashboard. It will be to keep a check on jobs applied for, interviews etc. The main thing I am concerned with at this point is the actual implementation, so that it does appear as a multi-value field (multidimensional array) per candidate record as a result of either a Plugin or ideally a Hybrid App implementation.

**Comment by gravitymaster97** [comment]
<a href="https://wordpress.org/plugins/custom-field-for-wp-job-manager/">Custom Field For WP Job Manager</a> using for add text field and select field in wordpress

**Comment by yscik** [comment]
Related: 7118687-zen
